readme.md

```diff
@@ -111,6 +111,8 @@ const curveEditor = new MojsCurveEditor({
   name:         'bounce curve'
   // if should preserve state on page reloads
   isSaveState:  true,
+  // if should draw some predefined start path, e.g.
```

Comment:
Let's change to something like this:
`start` path - it will be loaded on initialization of the `curve` - e.g. before any user modifications were made. Path of `M0, 100 L100, 0` by default.